CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.13.85](https://github.com/brave/brave-browser/releases/tag/v1.13.85)
+
+ - Implemented cross-platform promotion modal. ([#10571](https://github.com/brave/brave-browser/issues/10571))
+ - Implemented "Refresh Status" button on rewards panel to check for publisher status. ([#10005](https://github.com/brave/brave-browser/issues/10005))
+ - Implemented version 2 of onboarding. ([#9555](https://github.com/brave/brave-browser/issues/9555))
+ - Implemented fingerprint farbling for Plugins. ([#9435](https://github.com/brave/brave-browser/issues/9435))
+ - Implemented new wallet API for Brave Rewards. ([#8428](https://github.com/brave/brave-browser/issues/8428))
+ - Disabled ability to retrieve rewards logs via brave://rewards-internals and removed existing logs. ([#11260](https://github.com/brave/brave-browser/issues/11260))
+ - Fixed issue where ad conversions would fail when a site uses pushState. ([#11012](https://github.com/brave/brave-browser/issues/11012))
+ - Fixed crash issue when opening search settings. ([#10773](https://github.com/brave/brave-browser/issues/10773))
+ - Fixed an issue with opening downloaded PDF files. ([#11420](https://github.com/brave/brave-browser/issues/11420)
+ - Fixed sending links from Android to Desktop when sync is enabled. ([#11077](https://github.com/brave/brave-browser/issues/11077)
```

Comment:
Needs to describe what the issue was. Above doesn't describe what was broken.

* ` - Fixed not being able to send links from Android to Desktop when sync is enabled.`